docs/src/modules/brandingTheme.ts

```diff
@@ -288,9 +307,78 @@ export const getDesignTokens = (mode: 'light' | 'dark') =>
         scrollMarginTop: 'calc(var(--MuiDocs-header-height) + 32px)',
       },
     },
+    /**
+     * This utility exists to help transitioning to CSS variables page by page (prevent dark mode flicker).
```

Comment:
**Off-topic**

To consider if this could be exposed as a public API. We might not be the only ones that can't migrate all their codebase to CSS variables at once.

Reply:
Good point, I think we could expose a utility function but let's wait until we hear the need from the community.